src/app/carbon-estimator-form/carbon-estimator-form.component.html

```diff
@@ -134,78 +134,90 @@
   <div formGroupName="downstream">
     <ng-container *ngTemplateOutlet="sectionHeader; context: formContext.downstream"></ng-container>
     <div class="flex flex-col gap-4">
-      <div class="flex flex-col gap-2">
-        <div class="flex">
-          <label for="purposeOfSite" class="text-sm text-slate-600"
-            >What's the primary purpose of your digital services?</label
-          >
-          <helper-info class="ml-auto" inputLabel="What's the purpose of your website">
-            <p class="text-sm text-slate-600">
-              The purpose of your digital services is used to determine the typical amount of time a user might spend on
-              them and the amount of data that may be transferred.
-            </p>
-            <p class="text-sm text-slate-600">
-              This can affect the amount of energy used by end user devices and how much energy from network traffic is
-              attributable to your organisation
-            </p>
-          </helper-info>
-        </div>
-        <select
-          class="border border-slate-400 rounded"
-          formControlName="purposeOfSite"
-          name="purposeOfSite"
-          id="purposeOfSite">
-          <option value="average" selected>Unspecified (uses average)</option>
-          <option value="information">Information</option>
-          <option value="eCommerce">E-Commerce</option>
-          <option value="socialMedia">Social Media</option>
-          <option value="streaming">Streaming</option>
-        </select>
+      <div class="flex gap-2 items-center">
+        <input id="noDownstream" type="checkbox" formControlName="noDownstream" class="rounded-sm" />
```

Comment:
Was only a couple so I've just added that to the rest. Don't know if there's a way to avoid specifying on both? If I do it on the parent div it only shows to the right of the input/label.

Reply:
I dont think there is a way with tailwind